test/Microsoft.AspNetCore.Routing.Performance/RoutingBenchmark.cs

```diff
@@ -35,7 +35,7 @@ public RoutingBenchmark()
                 NullLoggerFactory.Instance,
                 UrlEncoder.Default,
                 new DefaultObjectPool<UriBuildingContext>(new UriBuilderContextPooledObjectPolicy(UrlEncoder.Default)),
-                new DefaultInlineConstraintResolver(new OptionsManager<RouteOptions>(Enumerable.Empty<IConfigureOptions<RouteOptions>>())));
+                new DefaultInlineConstraintResolver(new OptionsManager<RouteOptions>(new OptionsFactory<RouteOptions>(Enumerable.Empty<IConfigureOptions<RouteOptions>>(), Enumerable.Empty<IPostConfigureOptions<RouteOptions>>()))));
```

Comment:
It doesn't look THAT bad, WebAPIShim is the only one not using TestOptionsMonitor, which is how things are supposed to be used.